systemtest/src/main/java/io/strimzi/systemtest/resources/ResourceManager.java

```diff
@@ -163,7 +163,7 @@ public final <T extends HasMetadata> void createResourceWithWait(ExtensionContex
     }
 
     @SafeVarargs
-    private final <T extends HasMetadata> void createResource(ExtensionContext testContext, boolean waitReady, T... resources) {
+    private <T extends HasMetadata> void createResource(ExtensionContext testContext, boolean waitReady, T... resources) {
```

Comment:
Maybe I cannot see it in the code, but why did you changed it?

Reply:
Well, it is a `private` method, and when one marks a method with `final` then it says that it could not be implemented in the inheriting classes (but it's private so there is no need to use the `final` keyword here). Anyway, I think I forgot to change it during my refactor :/

Reply:
Ah okay, yeah. I thought that you need it somewhere or something.